common/src/main/java/com/wynntils/core/keybinds/KeyBind.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © Wynntils 2022.
+ * Copyright © Wynntils 2022-2023.
```

Comment:
I'm not sure what happened here, but this file has not been modified in 2023. 

I have fixed all correct updates to 2022-2023 in main and merged it here. I suggest you remove these manually; I don't think spotless will do it for you once it has put it in...